### DISCUSSION TITLE
Discussion page has been enabled

### DISCUSSION THREAD

**Discussion by Diman0** [Mailu/Mailu#3005] [Announcements]
👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. It will be used to

* List announcements
* Provide a place for general discussion.
* Place feature requests
* Ask for user support.

The issues section from now on is only meant for reporting bugs with clear replication steps. Use the discussion section for any other discussions and getting support.

**Comment by DerKnerd** [comment]
As stated in the issue that might have sparked this, I would like to contribute in the discussions, do you need mods or so?

**Reply by Diman0** [reply]
Thank you for the offer. For now that is not required. If it does get out of hand, then I might come back to you on your offer.

**Reply by DerKnerd** [reply]
Sounds good. I am happy to help, Mailu helped me a lot so I think it is justified to give something back.
